ui/service.go

```diff
@@ -65,17 +68,24 @@ func New(things mainflux.ThingsServiceClient, sdk sdk.SDK) Service {
 	}
 }
 
-func (gs *uiService) Index(ctx context.Context, token string) ([]byte, error) {
-	tpl := template.New("index")
+func ParseTemplate(tmp string) (*template.Template, error) {
```

Comment:
Is it used in other packages and has to be exported?

Reply:
Additionaly, it would be good to make this function variadic, so that it can accept more files as an input (some service handlers would potentinally like to pass several template files associated with them).